src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goBack(steps = 1) {
```

Comment:
since you use a singular value instead of a plural one. Apply for other cases as well
```suggestion
    goBack(step = 1) {
```